cmd/adapter/adapter.go

```diff
@@ -30,6 +30,7 @@ import (
 	basecmd "github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd"
 	"github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/provider"
 
+	customexternalmetrics "github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver"
```

Comment:
this import should be moved next to the other `custom-metrics-apiserver` imports

Reply:
Thanks @dgrisonnet. Fixed it.